qiskit/pulse/commands/__init__.py

```diff
@@ -27,7 +27,7 @@
 from .persistent_value import PersistentValue, PersistentValueInstruction
 from .command import Command
 from .pulse_decorators import functional_pulse
-from .sample_pulse import SamplePulse, PulseInstruction
+from .sample_pulse import Waveform, PulseInstruction
```

Comment:
```suggestion
from .sample_pulse import SamplePulse, PulseInstruction
```
The `commands/` are all deprecated. This is here to maintain backwards compatibility of this import path. Waveform shouldn't exist in `commands` anyway